oeis.py

```diff
@@ -382,6 +382,32 @@ def A000108(start: int = 0, limit: int = 20) -> Collection[int]:
         sequence.append(int(r))
 
     return sequence
+def fibonacci(n):
+    if n < 0:
+        print("Incorrect input")
+    if n <= 1:
+        return 1
+    else:
+        return fibonacci(n - 1) + fibonacci(n - 2)
+
+
+def A000045(start, limit, plot=False):
+
+    numbers = []
+    y_label = []
+    for i in range(int(start), int(limit) + 1):
```

Comment:
You probably mean:
```suggestion
    for i in range(int(start), int(start) + int(limit) + 1):
```

Else it's not behaving as a `limit` but as a `stop`.